src/network/tarcap/packet_types.hpp

```diff
@@ -0,0 +1,148 @@
+#pragma once
+
+#include <string>
+
+#include "common/types.hpp"
+
+namespace taraxa::network::tarcap {
+
+/**
+ * @brief SubprotocolPacketType is used in networking layer to differentiate packet types
+ */
+enum SubprotocolPacketType : uint32_t {
+  StatusPacket = 0,
+  NewBlockPacket,
+  NewBlockHashPacket,
+  GetNewBlockPacket,
+  GetBlocksPacket,
+  BlocksPacket,
+  TransactionPacket,
+  TestPacket,
+  PbftVotePacket,
+  GetPbftNextVotes,
+  PbftNextVotesPacket,
+  NewPbftBlockPacket,
+  GetPbftBlockPacket,
+  PbftBlockPacket,
+  SyncedPacket,
+  PacketCount
+};
+
+/**
+ * @brief PriorityQueuePacketType is mapped 1:1 to SubprotocolPacketType. It is used inside tarcap priority queue for
+ *        more efficient processing of different packets types dependencies thanks to it's bit mask representation
+ */
+enum PriorityQueuePacketType : uint32_t {
+  // Consensus packets with high processing priority
+  PQ_HighPriorityPackets = 0,
```

Comment:
why PQ_ prefix? it is against our style guide

Reply:
this enums maps 1 : 1 different enum with the same names inside so thats why the prefix

Reply:
isn't it better to used class enums then?

Reply:
could be enum class but there would be a lot of casting then as it is used for arithmetic operations in blocking mask...